### PR TITLE
Fix bugs with push-kibana errand

### PR DESCRIPTION
**Pull request by mrdavidlaing**
* Push to system (rather than app) domain
* Update Kibana OAuth client_secret during push-kibana errand

### CODE REVIEW

**Comment by mrdavidlaing** [comment]
https://logsearch-for-cloudfoundry-boshrelease.s3.amazonaws.com/boshrelease-logsearch-for-cloudfoundry-5%2Bdev.10.tgz

**Comment by mrdavidlaing** [comment]
https://logsearch-for-cloudfoundry-boshrelease.s3.amazonaws.com/boshrelease-logsearch-for-cloudfoundry-5%2Bdev.11.tgz

**Comment by mrdavidlaing** [comment]
https://logsearch-for-cloudfoundry-boshrelease.s3.amazonaws.com/boshrelease-logsearch-for-cloudfoundry-5%2Bdev.12.tgz

**Comment by mrdavidlaing** [comment]
https://logsearch-for-cloudfoundry-boshrelease.s3.amazonaws.com/boshrelease-logsearch-for-cloudfoundry-5%2Bdev.15.tgz

File: logsearch-for-cloudfoundry-boshrelease/templates/stub.yml [hunk 1]
```diff
@@ -20,18 +20,15 @@ properties:
     oauth2_client_id: "kibana_admin"     
     oauth2_client_secret: "keepitsimple" 
   redis:
-    host: 10.244.10.10
+    host: "10.244.11.10"
   cloudfoundry:
-    cloudControllerUri: "https://api.10.244.0.34.xip.io"
-    uaaUri: "https://uaa.10.244.0.34.xip.io"
+    system_domain: "workspace.pcf.stayup.io"
```
Comment (shinji62) [inline logsearch-for-cloudfoundry-boshrelease/templates/stub.yml]: sure about this one ?

Comment (mrdavidlaing) [inline logsearch-for-cloudfoundry-boshrelease/templates/stub.yml]: Oops.  The field names are correct.  But I shouldn't have committed my internal settings

**Comment by mrdavidlaing** [comment]
https://logsearch-for-cloudfoundry-boshrelease.s3.amazonaws.com/boshrelease-logsearch-for-cloudfoundry-5%2Bdev.18.tgz

**Pull request merged**